roxie/udplib/udptrs.cpp

```diff
@@ -204,7 +204,15 @@ class UdpReceiverEntry : public IUdpReceiverEntry
             {
                 if (encrypted)
                 {
-                    aesEncrypt(key, sizeof(key), buffer->data, length, encryptBuffer.clear());
+                    encryptBuffer.clear();
```

Comment:
could ensure capacity of DATA_PAYLOAD to avoid reallocation.

Reply:
Or length+16